extlinks/aggregates/management/commands/fill_link_aggregates.py

```diff
@@ -155,19 +154,15 @@ def _fill_link_aggregates(self, link_events, collection):
         None
         """
         for link_event in link_events:
-            if LinkAggregate.objects.filter(
+            link_aggregate = LinkAggregate.objects.filter(
                 organisation=collection.organisation,
                 collection=collection,
                 full_date=link_event["timestamp_date"],
                 on_user_list=link_event["on_user_list"],
-            ).exists():
+            )
+            if link_aggregate.exists():
                 # Query LinkAggregate for the existing field
-                existing_link_aggregate = LinkAggregate.objects.get(
-                    organisation=collection.organisation,
-                    collection=collection,
-                    full_date=link_event["timestamp_date"],
-                    on_user_list=link_event["on_user_list"],
-                )
+                existing_link_aggregate = link_aggregate.first()
```

Comment:
same suggestion as in `fill_pageproject_aggregates.py`

Reply:
Done